containers/verifier/install-dependencies.sh.j2

```diff
@@ -0,0 +1,69 @@
+#!/bin/bash -e
+
+# Update package lists for Debian and install dependencies
+dependencies=(
+    # Required by psycopg2 python package (PostgreSQL support)
+    libpq-dev
+
+    # Packages needed to build concent(python extensions, dependencies of golem-messages in particular)
+    libssl-dev
+    git
+    pkg-config
+    libffi-dev
+    software-properties-common
+
+    # Package needed to build golem verification
+    libglib2.0-0
+    libsm6
+    libxrender1
+    wget
+)
+
+testing_dependencies=(
+
+    # Packages needed to build python extensions, dependencies of golem-messages in particular
+    # They need to be installed from testing, because on Debian Jessie are too old or non-existent
+    binutils
+    build-essential
+    libsecp256k1-dev
+    python3-pip
+
+    # There's no stable Python 3.6 package in Debian Jessie so we have to get it from testing.
+    python3.6
+    python3.6-dev
+    virtualenv
+    python3-setuptools
+)
+
+apt-get update --assume-yes
+apt-get install              \
+    --assume-yes             \
+    --no-install-recommends  \
+    ${dependencies[*]}
+
+add-apt-repository "deb http://ftp.de.debian.org/debian testing main"
+apt-get update --assume-yes
+
+apt-get install              \
+    --target-release testing \
+    --assume-yes             \
+    --no-install-recommends  \
+    ${testing_dependencies[*]}
+
+
+# Get su-exec which will be used to change user id
+wget                                                                                                \
+    --output-document /tmp/su-exec "https://github.com/golemfactory/golem/wiki/binaries/su-exec"
+
+test "60e8c3010aaa85f5d919448d082ecdf6e8b75a1c  /tmp/su-exec" = "$(sha1sum /tmp/su-exec)"
+mv /tmp/su-exec /usr/local/bin/su-exec
+chmod +x /usr/local/bin/su-exec
+su-exec nobody true
+
+# Install dependencies needed by golem verification
+pip3 install --no-cache-dir --requirement /golem/work/requirements.txt
```

Comment:
Yeah. We're adding them because we're going to use them in our verification. Golem's verifier is not ready for use in Concent anyway. So they're going to be used by both.

Reply:
Done.